### ISSUE TITLE
Window tests fail on WIndows 10

### ISSUE THREAD

**Issue by jkvatne** [fyne-io/fyne#1037]
### Describe the bug:
A lot of the tests in window_test.go and canvas_test.go fails (depending on screen resolution)
The problems mostly seems to have a single cause: The window size can not be set small enough. When the window is decorated, the minimum width will be >=120, while the tests use smaller windows. The minimum width will depend on screen resolution and scaling.

Changing the createWindow(title) code setting w.decorate=false will mostly fix the problems. But of cause the test for decorate=true will fail, and either createWIndow must have a "decorate" parameter, or the test must be changed.

### Example code:
```
func createWindow(title string) fyne.Window {
	w := d.CreateWindow(title)
	w.(*window).decorate = false
	w.(*window).create()
	return w
}
```
### Device (please complete the following information):
 - OS: Windows 10
 - Go version: 1.14.2
 - Fyne version: develop branch, sha=89e2b68764088c

**Comment by andydotxyz** [comment]
Thanks for this, it had not occurred to me that scale would impact the tests in this way. I will turn scale down to 100% next time I run the suite on Windows and see where that gets me.

**Comment by jkvatne** [comment]
It was not enough to set scaling to 100%. It still failed.

**Comment by andydotxyz** [comment]
Apoloiges, I did not mean that as a fix - I meant that becuse I am at 200% I have missed seeing test failures, so I will try to replicate by putting scale down on my Windows install.

**Comment by andydotxyz** [comment]
This should be fixed by ac1a0a2, on develop for testing
